modules/javafx.controls/src/main/java/javafx/scene/control/skin/ChoiceBoxSkin.java

```diff
@@ -364,6 +343,11 @@ String getChoiceBoxSelectedText() {
         return label.getText();
     }
 
+    // Test only purpose
+    ContextMenu getChoiceBoxPopup() {
```

Comment:
well, as TabPaneSkin is a singularity in controls (in graphics such a pattern seems to be wider-spread), I would prefer not to do it here - also because ChoiceBoxSkin has another test-only accessor (for label text) which doesn't. Might be a candidate for a general cleanup task - if we decide to really introduce such a naming convention (which I personally wouldn't like :)